src/python/perfflowaspect/advice_chrome.py

```diff
@@ -225,6 +225,8 @@ def create_async_event(cls, name, cat, ph, scope=None, id=None):
         event["ph"] = ph
         if id:
             event["id"] = id
+        else:
+            event["id"] = "123" #Async events need an ID, this sets the default.
```

Comment:
Does this need to be a string or should it be an int (no quotes)?

Reply:
Good question, I don't know if it matters, but looks like we can go with an `int`. I had picked that up from Dong's code somewhere I think. 

Looking at the format doc, Page 8, they seem to be using a hex ID: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#